service/src/main/webapp/course/file.jsp

```diff
@@ -0,0 +1,52 @@
+<%@ page language="java" contentType="text/html; charset=UTF-8" pageEncoding="UTF-8"
+    import="java.util.*, com.fecklessweasel.service.*, com.fecklessweasel.service.objectmodel.*, com.fecklessweasel.service.datatier.*, java.sql.Connection"%>
+<html>
+    <head>
+        <title>Feckless Weasel File Viewer</title>
+        <link href="${pageContext.request.contextPath}/assets/bootstrap-3.3.5-dist/css/bootstrap.min.css" rel="stylesheet">
+    </head>
+        <body class="four-column">
+            <jsp:include page="/header.jsp"/>
+            <%
+                final UserSession authSession = UserSessionUtil.resumeSession(request);
+                Tuple3<StoredFile, Course, User> tuple =
+                SQLSource.interact(new SQLInteractionInterface<Tuple3<StoredFile, Course, User>>() {
```

Comment:
im pretty sure we decided to store object references as the actual objects rather than by ids. So I think stored file needs to be changed to hold the actual class and user. That should eliminate the need for the tuples i think.

Right now on master the stored file object has

/** The user id of the user who uploaded the file. */
    private int uid;
    /** The id of the course that the file pertains to */
    private int cid;